src/options.ts

```diff
@@ -115,3 +124,13 @@ export function defaultSpanExporterFactory(options: Options): JaegerExporter {
 export function defaultSpanProcessorFactory(options: Options): SpanProcessor {
   return new BatchSpanProcessor(options.spanExporterFactory(options));
 }
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function defaultPropagatorFactory(options: Options): TextMapPropagator {
+  return new CompositePropagator({
+    propagators: [
+      new B3Propagator({ injectEncoding: B3InjectEncoding.MULTI_HEADER }),
```

Comment:
Just to educate myself: is b3 multi the default propagator for Splunk distros? And why are both used?

Reply:
>is b3 multi the default propagator for Splunk distros

Yes

Reply:
> And why are both used?

In order to be compatible with both SFX libs and any other non-Splunk (or differently configured) Otel libs, and provide an easier migration path I'd say.